### PR TITLE
fix: Make the application of an ulimit to open fds configurable.

### PR DESCRIPTION
**Pull request by andreas-kupries**
New property `set_ulimit_fds` to disable ulimiting.
Defaults to true, activating the ulimit for standard behaviour.

The change allows SUSE to disable the application of the limit in their containerization (where it interferes) without having to patch system sources. After the change this can be done properly via ops files changing the relevant properties.

Origin ticket: cloudfoundry-incubator/kubecf#504
Origin ticket: cloudfoundry-incubator/kubecf#505
See also https://github.com/cloudfoundry/diego-release/pull/512

* [x] I have viewed signed and have submitted the Contributor License Agreement

* [x] I have made this pull request to the `develop` branch

* [ ] I have run CF Acceptance Tests on bosh lite

### CODE REVIEW

File: jobs/cc_uploader/spec [hunk 1]
```diff
@@ -29,6 +29,10 @@ provides:
     - https_port
 
 properties:
+  set_ulimit_fds:
```
Comment (cwlbraa) [inline jobs/cc_uploader/spec:32]: please namespace this under `capi.cc_uploader.set_ulimit_fds` and move it lower in the properties hash.

File: jobs/cloud_controller_ng/spec [hunk 2]
```diff
@@ -253,6 +253,10 @@ consumes:
   optional: true
 
 properties:
+  set_ulimit_fds:
```
Comment (cwlbraa) [inline jobs/cloud_controller_ng/spec:256]: likewise, please namespace this under `cc.set_ulimit_fds` and move it down

**Comment by viovanov** [comment]
@andreas-kupries can you please address @cwlbraa 's comments?

**Comment by andreas-kupries** [comment]
@viovanov Yes. Apologies for missing them when they were made.

File: jobs/cc_uploader/spec (hunk 1 above)
Comment (andreas-kupries) [reply jobs/cc_uploader/spec]: Done with commit 3bc3b594c92e6f3b054ea16dd1060593ee7020f1.

File: jobs/cloud_controller_ng/spec (hunk 2 above)
Comment (andreas-kupries) [reply jobs/cloud_controller_ng/spec]: Done with commit 3bc3b594c92e6f3b054ea16dd1060593ee7020f1.

**Comment by andreas-kupries** [comment]
Sigh. Embarassing.
The relevant files are old-style job control scripts.
They are not used by the cf-operator based kubecf, anymore, at al.
The patch is useless, and bringing this upstream is nonsense.

See https://github.com/cloudfoundry-incubator/kubecf/pull/1040 for the PR which has now removed these patches from kubecf itself.

Closing with apologies.

**Pull request closed without merging**